app/views/feedback/new.html.erb

```diff
@@ -1,44 +1,57 @@
-<%= title "Submit Feedback" %>
+<%= title "Help" %>
```

Comment:
I like the new copy for this page 👍 

But I think the route should also be changed to `/help` instead of `/feedback`.